components/LinkMoreRoutes.vue

```diff
@@ -0,0 +1,31 @@
+<script setup>
+const props = defineProps({
+  name: {
+    type: String,
+    required: true
+  },
+  href: {
+    type: String,
+    required: true
+  },
+  rel: {
+    type: String,
+    default: 'noopener noreferrer'
+  }
+});
+</script>
+
+<template>
+   <v-btn :href="href" :rel="rel" block variant="flat" rounded="lg" density="default" class="text-none mt-6"  >
+      {{ name }}
+   </v-btn>
+ </template>
+
+<style scoped>
+.v-btn {
+  background: var(--grey-backgroud);
+  letter-spacing: 0.1px;
```

Comment:
@PiotrDrechsler widzę, że co najmniej letter-spacing i line height się nam powtórzyły - może warto już to wyciągnąć do osobnego pliku jako zmienne css-owe, analogicznie do _colors.css? Albo skorzystać z sass i jego mixinów (vuetify chyba w domyśle ma go ze sobą)?

Reply:
Ok. Dodałem plik ze zmiennymi _vars. Nazwałem je adekwatnie to używanego komponentu.